pkg/scaffold/v2/certmanager/kustomize.go

```diff
@@ -39,13 +39,22 @@ func (p *Kustomization) GetInput() (input.Input, error) {
 var kustomizationTemplate = `resources:
 - certificate.yaml
 
+# the following config is for teaching kustomize how to do var substitution
 vars:
 - name: CERTIFICATENAME
   objref:
     kind: Certificate
     group: certmanager.k8s.io
     version: v1alpha1
     name: serving-cert # this name should match the one in certificate.yaml
+- name: CERTIFICATENAMESPACE
```

Comment:
CERTIFICATE_NAMESPACE ? to improve readability
It will also be good to mention why do we need this to be var ?